apps/design/backend/src/features.ts

```diff
@@ -12,53 +12,70 @@ export function isVxOrSliOrg(orgId: string): boolean {
  * officials can do.
  */
 export enum UserFeature {
```

Comment:
No functional change in this commit (https://github.com/votingworks/vxsuite/pull/7396/commits/3019d8897503613f20a3e18da152e58f3493e444)

Just rearranging and renaming some feature flags, grouping all feature flags involving the export screen and system settings screen